suite-native/navigation/src/navigators.ts

```diff
@@ -42,15 +48,15 @@ export type SettingsStackParamList = {
     [SettingsStackRoutes.SettingsFAQ]: undefined;
 };
 
-export type SendReceiveStackParamList = {
-    [SendReceiveStackRoutes.ReceiveAccounts]: undefined;
-    [SendReceiveStackRoutes.Receive]: { accountKey: AccountKey; tokenSymbol?: TokenSymbol };
+export type ReceiveStackParamList = {
+    [ReceiveStackRoutes.ReceiveAccounts]: ReceiveAccountsParams;
```

Comment:
Why `ReceiveAccounts` need these parameters? You do not access them in `ReceiveAccountsScreen` IMO should be `undefined`.

Reply:
https://github.com/trezor/trezor-suite/pull/8316/commits/eaebbfb6a11e30414886e378d6cc6708f05b6ca5